Makefile

```
@@ -6,8 +6,8 @@ export IMAGE_REPO ?= quay.io/operator-framework/operator-controller
 export IMAGE_TAG ?= devel
 export GO_BUILD_TAGS ?= upstream
 export CERT_MGR_VERSION ?= v1.9.0
+export CATALOGD_VERSION ?= v0.1.2
```

```suggestion
export CATALOGD_VERSION ?= v0.1.3
```

New catalogd version released with fixes to the image architectures